index.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+require_once 'src/Services/DatabaseService.php';
+require_once 'src/Models/ProjectModel.php';
+require_once 'src/Services/ProjectDisplayService.php';
+
+$db = DatabaseService::connect();
+
+$projectModel = new ProjectModel($db);
+
+$projects = $projectModel->getProjects();
+
+?>
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Project Manager</title>
+    <script src="https://cdn.tailwindcss.com"></script>
+</head>
+<body>
+
+    <header class="p-3 bg-teal-50 flex justify-between">
+        <h1 class="sm:text-5xl text-4xl"><a href="index.html">Project Manager</a></h1>
+    </header>
+    <main class="p-3">
+        <h2 class="text-4xl font-bold mb-2">Projects</h2>
+<!--        <h4 class="text-2xl">Filter projects by Client:</h4>-->
+<!--        <form>-->
+<!--            <select class="px-3 py-1 rounded border">-->
+<!--                <option>All Clients</option>-->
+<!--                <option>Client 1</option>-->
+<!--                <option>Client 2</option>-->
+<!--                <option>Client 3</option>-->
+<!--            </select>-->
+<!--            <input type="submit" value="Apply" class="rounded bg-green-100 px-3 py-1">-->
+<!--        </form>-->
```

Comment:
Please remove this commented out chunk - you may not ever get to this story, so we don't want to leave this here. Commented out HTML does still get downloaded by the web browser, so this will be (in a tiiiiiiny way) slowing down the page load speed

Reply:
rudeness LVL 8 but done ✅